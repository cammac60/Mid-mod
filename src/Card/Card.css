.card-wrapper {
  background-color: white;
  border: 2px solid teal;
  height: 200px;
  text-align: center;
  width: 100%;
}

#date, #time, #guests {
  display: block;
}
