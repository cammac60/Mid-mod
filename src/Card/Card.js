import React from 'react';
import './Card.css';

const Card = (props) => {
  return (
    <div className="card-wrapper">
      <h4>{props.name}</h4>
      <span id='date'>{props.date}</span>
      <span id='time'>{props.time}</span>
      <span id='guests'>Number of guests: {props.number}</span>
    </div>
  )
}

export default Card;
