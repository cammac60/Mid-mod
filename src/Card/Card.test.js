import React from 'react';
import ReactDOM from 'react-dom';
import Card from './card';
import { shallow } from 'enzyme';

describe('Card', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Card />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  
});
