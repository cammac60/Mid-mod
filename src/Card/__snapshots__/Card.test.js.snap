// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div
  className="card-wrapper"
>
  <h4 />
  <span
    id="date"
  />
  <span
    id="time"
  />
  <span
    id="guests"
  >
    Number of guests: 
  </span>
</div>
`;
